.github/workflows/ci.yaml

```diff
@@ -77,10 +78,10 @@ jobs:
           nix build ./#containers.x86_64-linux.cartservice.arm64 --no-link --print-out-paths
           nix build ./#containers.x86_64-linux.cartservice.amd64 --no-link --print-out-paths
 
-      - name: Build currencyexternalapi images
+      - name: Build metrics images
         run: |
-          nix build ./#containers.x86_64-linux.currencyexternalapi.arm64 --no-link --print-out-paths
-          nix build ./#containers.x86_64-linux.currencyexternalapi.amd64 --no-link --print-out-paths
```

Comment:
it isn't necessary to build an image for the  `currencyexternalapi` package because it's a library used in other services